mava/components/training/step.py

```diff
@@ -478,8 +495,27 @@ def step(sample: reverb.ReplaySample) -> Tuple[Dict[str, jnp.ndarray]]:
                     constants.OPT_STATE_DICT_KEY
                 ] = new_states.critic_opt_states[net_key][constants.OPT_STATE_DICT_KEY]
 
-            # update the running stats
-            trainer.store.stats = new_states.stats
+            # Update the observation normalization parameters
+            for agent in trainer.store.trainer_agent_net_keys.keys():
+                obs_shape = len(
+                    trainer.store.obs_norm_params[constants.OBS_NORM_STATE_DICT_KEY][
```

Comment:
Maybe create a variable for  `trainer.store.obs_norm_params[constants.OBS_NORM_STATE_DICT_KEY][agent]` to make it easier to read.

Reply:
Just made a PR to make this easier to read. I made a variable for constants.OBS_NORM_STATE_DICT_KEY.

Reply:
Thanks :slightly_smiling_face: I was wondering if you could not make the variable for the entire `trainer.store.obs_norm_params[constants.OBS_NORM_STATE_DICT_KEY][agent]` instead, inside the for loop. But its up to you.